include/os.h

```diff
@@ -122,6 +122,8 @@ typedef CARD32 (*OsTimerCallback) (OsTimerPtr timer,
                                    CARD32 time,
                                    void *arg);
 
+extern _X_EXPORT Bool TimerForce(OsTimerPtr);
```

Comment:
should not go back into public header (thus becoming part of public API) again.
just _X_EXPORT the symbol and add a comment that this just for Nvidia backwards (for legacy 390).